frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/PiplelineWorkspacesSection.tsx

```diff
@@ -36,6 +38,20 @@ const getVolumeTypeFields = (volumeType: VolumeTypes, index: number, t: TFunctio
         />
       );
     }
+    case VolumeTypes.EmptyDirectory: {
+      return (
+        <Alert variant="info" title="this option could not be used to share data between tasks" />
```

Comment:
```suggestion
        <Alert inline variant="info" title="this option could not be used to share data between tasks" />
```
We don't use non-inline versions anywhere in the console I don't think. At least not in the DevConsole (that I have seen)

We'll also need to translate this... but I suppose we can wait on UX wording.

Reply:
And the wording has been figured out [here on the designs](https://docs.google.com/document/d/12ZETv528Puq41o9zg8_GCvUhhd5g8NEiGbA3bnmawjs/edit?disco=AAAALzTmAK4).

Reply:
updated